lib/config/types.ts

```diff
@@ -93,6 +93,7 @@ export interface RepoAdminConfig {
   dockerUser?: string;
   dryRun?: boolean;
   exposeAllEnv?: boolean;
+  gitNoVerify?: ('commit' | 'push')[];
```

Comment:
Exctrat a type and reuse, eg `export type GitNoVerifyOption = 'commit' | 'push';`

Reply:
I put the extracted type in this file, although it does feel like it should belong in the git file instead since the function should own its parameters. Open to moving if you agree, otherwise I'll leave it in here.